webxr-api/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "webxr-api"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
This repo is only used as a git dependency so the version does not need to change.